src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Projects/NetCorePackageReferenceProjectProvider.cs

```diff
@@ -94,13 +103,16 @@ public NetCorePackageReferenceProjectProvider(IProjectSystemCache projectSystemC
             var fullProjectPath = vsProject.FullProjectPath;
             var unconfiguredProject = GetUnconfiguredProject(vsProject.Project);
 
+            var projectServices = new VsCoreProjectSystemServices(vsProject, _componentModel.Value);
```

Comment:
Although it fixes the issue `VsCoreProjectSystemServices` can't be used with net core project due to use of `DTE`. Better fix would be creating new services class providing script service only.